### PR TITLE
amo be gone (bug 1042820)

### PR DESCRIPTION
**Pull request by eviljeff**
Touch all the files!
note: this was a find/replace so there will probably be some messy import lines.  Tests pass and it (trivially) runs though.

### CODE REVIEW

File: mkt/site/log.py [hunk 1]
```diff
@@ -334,8 +334,8 @@ class DELETE_REVIEW(_LOG):
     """Requires review.id and add-on objects."""
```
Comment (eviljeff) [inline mkt/site/log.py]: I will tidy log.py after, promise :smile:

File: mkt/site/mail.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 from mkt.zadmin.models import get_config
 
 
-maillog = logging.getLogger('z.amo.mail')
+maillog = logging.getLogger('z.mkt.mail')
```
Comment (eviljeff) [inline mkt/site/mail.py]: this is probably a mistake replace

**Comment by robhudson** [comment]
Can you rebase so tests can run?

**Comment by robhudson** [comment]
`docs/topics/translations.rst` has an `import amo` in it.

File: mkt/__init__.py [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
+from mkt.constants.platforms import *
+from mkt.constants.search import *
 from mkt.constants.submit import *
+
+# This is used in multiple other files to access logging, do not remove.
+from mkt.site.log import (_LOG, LOG, LOG_BY_ID, LOG_ADMINS, LOG_EDITORS,
+                          LOG_HIDE_DEVELOPER, LOG_KEEP, LOG_REVIEW_QUEUE,
+                          LOG_REVIEW_EMAIL_USER, log)
+
+logger_log = commonware.log.getLogger('z.amo')
```
Comment (robhudson) [inline mkt/__init__.py]: `z.mkt` for the logger?

File: mkt/developers/forms.py [hunk 4]
```diff
@@ -26,10 +26,10 @@
 from quieter_formset.formset import BaseModelFormSet
 from tower import ugettext as _, ugettext_lazy as _lazy, ungettext as ngettext
 
-import amo
+import mkt
 import lib.iarc
 import mkt
```
Comment (robhudson) [inline mkt/developers/forms.py]: `import mkt` is here twice.

**Comment by robhudson** [comment]
r+wc. This looks good. Let's merge early so we can get a week of testing on this on -dev.

**Pull request merged**